src/ui/Windows/StateBitmapControl.cpp

```diff
@@ -0,0 +1,220 @@
+/*
+* Copyright (c) 2003-2023 Rony Shapiro <ronys@pwsafe.org>.
+* All rights reserved. Use of the code is allowed under the
+* Artistic License 2.0 terms, as specified in the LICENSE file
+* distributed with this code, or available from
+* http://www.opensource.org/licenses/artistic-license-2.0.php
+*/
+
+#pragma once
```

Comment:
"#pragma once" doesn't really make sense for .cpp files:
ExcludeCaptureStateBitmapManager.cpp
ScreenCaptureStateControl.cpp
StateBitmapControl.cpp
StateBitmapManager.cpp

Reply:
> "#pragma once" doesn't really make sense for .cpp files: ExcludeCaptureStateBitmapManager.cpp ScreenCaptureStateControl.cpp StateBitmapControl.cpp StateBitmapManager.cpp

So true... so so true! 💡